chargebee/src/main/java/com/chargebee/android/billingservice/BillingClientManager.kt

```diff
@@ -253,6 +255,102 @@ class BillingClientManager(context: Context) : PurchasesUpdatedListener {
 
     }
 
+    internal fun changeProduct(
+        changeProductParams: ChangeProductParams,
+        purchaseCallBack: CBCallback.PurchaseCallback<String>
+    ) {
+        this.purchaseCallBack = purchaseCallBack
+        onConnected({ status ->
+            if (status) {
+                changeProduct(changeProductParams)
+            } else
+                purchaseCallBack.onError(
+                    connectionError
+                )
+        }, { error ->
+            purchaseCallBack.onError(error)
+        })
+
+    }
+
+    private fun changeProduct(changeProductParams: ChangeProductParams) {
+        this.changeProductParams = changeProductParams
+        this.purchaseProductParams = changeProductParams.purchaseProductParams
```

Comment:
Do we need to set this?

Reply:
Yes its used in acknowledgePurchase method